### DISCUSSION TITLE
Cannot deploy static Next js inside a seperate folder

### DISCUSSION THREAD

**Discussion by dotconnor** [vercel/vercel#4043] [General]
My project structure looks like this:

```
src/
lib/
docs/
  pages/
  next.config.js
package.json
```

I run `next build docs && next export docs` to create the folder `docs/out/`, but the deployment always fails with `A routes-manifest.json couldn't be found.`

I have tried changing the output directory on the dashboard to `docs/`, `docs/out/`, `docs/.next/` but they all fail.

I would change the root directory in the dashboard but the site depends on the code in the root directory and there isn't a separate `package.json` for the docs.

**Comment by mcsdevv** [comment]
Hey @dotconnor, can you provide the source code for this so that we can reproduce and take a look?

**Reply by dotconnor** [reply]
Here's a minimal repro: https://github.com/dotconnor/next-test.
From what i can tell, the system always detects next.js and tries to find the `.next` folder but I'm just trying to serve the static files from `next export`

**Reply by mcsdevv** [reply]
Hey @dotconnor, thanks for providing that. We always recommend building Next.js on the platform as it gets handled a little differently from other frameworks. Can you tell me why you're building before deployment?

**Reply by dotconnor** [reply]
Sorry for the confusion, I am running next build and next export during the deployment.
